### PR TITLE
Update template dependencies

### PR DESCRIPTION
**Pull request by jcvalerio**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,95 +29,104 @@
...
+    "@reduxjs/toolkit": "1.4.0",
+    "@semantic-release/changelog": "5.0.1",
+    "@semantic-release/commit-analyzer": "8.0.1",
+    "@semantic-release/git": "9.0.0",
+    "@semantic-release/release-notes-generator": "9.0.1",
+    "apollo-datasource": "0.7.1",
+    "apollo-datasource-rest": "0.9.2",
+    "apollo-server-express": "2.15.0",
+    "aws-amplify-react": "4.1.17",
+    "aws-sdk": "^2.707.0",
```
Comment (jorgekleeen) [inline package.json:58]:
```suggestion
    "aws-sdk": "2.707.0",
```

File: package.json [hunk 2]
```diff
@@ -29,95 +29,104 @@
...
+    "jest": "26.1.0",
+    "prettier": "2.0.5",
+    "serverless-dotenv-plugin": "2.4.2",
+    "serverless-finch": "2.6.0",
+    "ts-jest": "26.1.1",
+    "ts-node": "8.10.2",
+    "tslint": "6.1.2",
+    "typescript": "3.9.5",
+    "webpack-cli": "3.3.12",
+    "eslint-plugin-cypress": "^2.11.1"
```
Comment (jorgekleeen) [inline package.json:129]: Can you install this again through npm so it locate it properly?
```suggestion
    "eslint-plugin-cypress": "2.11.1"
```

File: .eslintrc [hunk 3]
```diff
@@ -12,19 +12,49 @@
...
+      1,
+      {
+        "allow": ["warn", "error"]
+      }
+    ],
+    "no-shadow": 1,
+    "max-len": [1, 110],
+    "no-var": 2,
+    "sort-imports": 1,
+    "@typescript-eslint/camelcase": [1],
```
[the remark this answers is not in the document]
Comment (jcvalerio) [reply .eslintrc]: changes applied

File: package.json (hunk 1 above)
Comment (jcvalerio) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (jcvalerio) [reply package.json]: done 👍

**Pull request merged**